scripts/build.mjs

```diff
@@ -1,3 +1,5 @@
+/* c8 ignore start */
```

Comment:
Can vite not be configured to ignore other directories than `src`?

Reply:
Ah, sry should be an artifact